Lambda-School-Labs/CS10-quizzer
# Abbott/client classes page

# Description

Most of the work (minus styling) for adding classes and students is functional:
- Can create new classes
- Class list will display students in a class, as well as quizzes (though no quizzes can be added to a class through the client yet)
- Can change the name of a created class, titles will update on the spot
- Can add and remove students to a class, class list will update on the spot

## Type of change

Please delete options that are not relevant.


# How Has This Been Tested?

- Launch the server and the test client (or, alternatively, build the client and host it from Django)
- Create a new account or sign into an existing one
- Navigate to Classes:
   - If you have created any classes directly through GraphiQL, they should show up here automatically.
- Click the (screwy looking right now) Add Class button
   - This will pop up a modal window for you to enter the name of the new class.
   - Once a name is entered and submitted, you will likely see the new class pop up in the list behind the modal (later this should auto-redirect to the class edit) - close the modal and click on the newly created class
- From here, you can: 
   - Change the name of the class if desired (enter a new name into the field and hit enter, it will update automatically.
   - Add new students to the class by entering their names and emails. Once added to the database, they should appear in the list automatically.
   - Click on any existing student for a dropdown menu, which can be used to delete the student from the class (there are no quiz scores to view yet). Deleting a student will pop a modal verifying your decision, and deleting the student should update the list on the spot.

# Checklist:
